components/raceDetails.js

```diff
@@ -29,24 +48,32 @@ export default class RaceList extends React.Component {
         const {race, refreshing} = this.state;
 
         return (<View style={style}>
-            <Header
-              centerComponent={{ text: 'XCZLD', style: { color: '#fff' } }}
-              backgroundColor="#009688"
-            />
             <Text h2 style={{textAlign: 'center'}}>
               {race && race.name}
             </Text>
+            <Text h4 style={{ marginTop: 10, marginLeft: 20 }}>
+                Natjecatelji
+                <Text style={{ color: '#ff5252' }}> {race.race_results && race.race_results.length}</Text>
+            </Text>
             <List>
-                <FlatList
-                  keyExtractor={(item, index) => item.id}
-                  renderItem={({item})=>(<ListItem
+                <SectionList
+                    keyExtractor={(item, index) => item.id}
+                    ListEmptyComponent={<Text> Ucitavam... </Text>}
+                    refreshing={refreshing}
+                    onRefresh={this.loadRace}
+                    sections={this.getSectionsWithData()}
+                    renderSectionHeader={({section}) => (<ListItem
+                      titleStyle={{ fontWeight: 'bold', fontSize: 22 }}
+                      title={section.title}
+                      hideChevron={true}
+                    />)}
+                    renderItem={({item})=>(<ListItem
```

Comment:
Maybe some `itemRenderer` function call would come in handy? It's hard to read what component renders from all the code.